Marlin/src/pins/stm32f1/pins_MKS_ROBIN_PRO.h

```diff
@@ -403,3 +403,16 @@
   #define SPI_FLASH_MISO_PIN                PB14
   #define SPI_FLASH_MOSI_PIN                PB15
 #endif
+
+// MKS WIFI pins
+  #define WIFI_RESET_PIN                    PA5
+  #define WIFI_SERIAL                        1  // USART1
+  #define WIFI_IO1_PIN                      PC7
+  #define WIFI_IO0_PIN                      NC // N.C. 
```

Comment:
Looking at the other MKS boards with these pins, they are conditional based on `HAS_TFT_LVGL_UI` or `ENABLED(MKS_WIFI_MODULE)`.

Reply:
-1 make more compiling warnings as NC should i do that ?

Reply:
> -1 make more compiling warnings as NC should i do that ?

What kind of warnings?

Reply:
C:\Users\fred\.platformio\packages\framework-arduinoststm32\cores\arduino\stm32/PinNamesTypes.h:130:22: warning: overflow in conversion from 'uint32_t' {aka 'long unsigned int'} to 'PinName' changes value from '4294967295' to 'NC' [-Woverflow]

Reply:
i have find this about MKS_WIFI :
#define WIFI_SERIAL                       1  // USART1
  #define MKS_WIFI_MODULE_SERIAL            1  // USART1
  //#define MKS_WIFI_MODULE_SPI             2  // not connected

a double déclaration of wifi serial and unused SPI port should i add  this too ?
On the shematics the wifi module is not connected to spi but for modified boards it's could be cool to have the syntax

Reply:
Does WiFi still work if you leave it in, but comment it out?

```c
//#define WIFI_IO0_PIN                    -1 // NC
```

Reply:
the wifi works well with the actual imported modifications, i will test with spi port active but it's not wired on the mobo.

Reply:
We long ago considered replacing "-1" with "`NC`" or "`NOPIN`" (so we could do code inlining of `PIN_EXISTS`) but it wasn't in the cards. Ironically it's because `NC`/`NOPIN` is often defined as -1. We could use lowercase "`nc`" or "`nopin`" while making sure that these macros are not defined … but that's something for another day.